charts/redpanda/values.go

```diff
@@ -411,9 +411,10 @@ type PandaProxyClient struct {
 
 type TLSCert struct {
 	// Enabled   bool   `json:"enabled"`
-	CAEnabled bool   `json:"caEnabled" jsonschema:"required"`
-	Duration  string `json:"duration" jsonschema:"pattern=.*[smh]$"`
-	IssuerRef struct {
+	CAEnabled             bool   `json:"caEnabled" jsonschema:"required"`
+	Duration              string `json:"duration" jsonschema:"pattern=.*[smh]$"`
+	ApplyInternalDNSNames bool   `json:"applyInternalDNSNames"`
```

Comment:
nit: How about `includeInternalDNSNames`? I don't think `Apply` has a very understandable meaning in this context, do you?

Reply:
Include would mean add in addition to. Apply means to place. I think apply makes more sense than include since no additional dnsNames are added.

Reply:
Aren't the external DNS names "added" to the list? lines 54-57 in certs.yaml (Can't link to non-diffs in a PR :( )

We could also consult our user facing language expert @JakeSCahill ?